google-cloud-logging/src/main/java/com/google/cloud/logging/Logging.java

```diff
@@ -597,6 +597,57 @@ public static TailOption project(String project) {
    */
   boolean deleteLog(String log);
 
+  /**
+   * Deletes a log and all its log entries for given log destination (see 'logName' parameter in
+   * https://cloud.google.com/logging/docs/reference/v2/rest/v2/LogEntry). The log will reappear if
+   * new entries are written to it.
+   *
+   * <p>Example of deleting a log.
+   *
+   * <pre>
+   * {
+   *   &#64;code
+   *   String logName = "my_log_name";
+   *   String folder = "my_folder";
```

Comment:
nit: worth to emphasis these are folder *ID* and log *ID* to compose the qualified log name.